Makefile

```diff
@@ -1,44 +1,58 @@
-.PHONY: build install uninstall clean nbext-deps install-nbext uninstall-nbext labext-deps install-labext uninstall-labext
+.PHONY: install uninstall build clean publish
 
+
+# Build and installation
 install:
-	pip install -e .
+	pip install -v -e .
+	pip install -r requirements-dev.txt
 
 uninstall:
 	pip uninstall higlass-python
 
+postinstall: nbext-deps nbext-install labext-deps labext-install
+
 build:
 	python setup.py jsdeps
 
-clean:
+clean-py:
+	rm -rf build/
+
+clean-js:
 	rm -rf higlass/static/
```

Comment:
this seems to be unrelated to javascript or am I missing something? it probably doesn't hurt though

Reply:
`higlass/static/` is where the final extension js is dumped that ships with `higlass-python`; `build/` is where everything goes when publishing a source tarball of the python package